src/components/EstimatedLifecycleCost/Review/index.tsx

```diff
@@ -30,64 +30,80 @@ const EstimatedLifecycleCostReview = ({
     year4: 'Year 4',
     year5: 'Year 5'
   };
-  const costForPhase = (items: LifecyclePhase[], phaseID: string) => {
-    const phaseMap: { [key: string]: string } = {
-      development: 'Development',
-      om: 'Operations and Maintenance',
-      other: 'Other'
-    };
-
-    const phase = items.find(obj => obj.phase === phaseMap[phaseID]);
-    if (phase) {
-      return parseFloat(phase.cost);
-    }
-    return undefined;
-  };
 
   const formatDollarsOrDash = (value: number | undefined): string => {
-    if (typeof value === 'undefined') {
+    if (Number.isNaN(value)) {
       return '-';
     }
     return formatDollars(value);
   };
 
-  const sum = (values: (number | undefined)[]): number => {
+  const sum = (values: number[]): number => {
```

Comment:
Instead of opening the type to accept `undefined`, I made it only `number`.

`NaN` is actually a number(e.g. `typeof NaN === 'number'`)